taxa/profile/tpeditor.php

```diff
@@ -188,11 +188,12 @@ function openOccurrenceSearch(target) {
 		?>
 	</div>
 	<div id="innertext">
+		<h1 class="page-heading"><?php echo $LANG['TAXON_EDITOR'] .': ' . $tEditor->getSciName(); ?></h1>
 		<?php
 		if($tEditor->getTid()){
 			if($isEditor){
 				if($tEditor->isForwarded()) echo '<div id="redirectedfrom">'.$LANG['REDIRECTED_FROM'].': <i>'.$tEditor->getSubmittedValue('sciname').'</i></div>';
-				echo '<div id="taxonDiv"><a href="../index.php?taxon=' . htmlspecialchars($tEditor->getTid(), HTML_SPECIAL_CHARS_FLAGS) . '">' . htmlspecialchars($tEditor->getSciName(), HTML_SPECIAL_CHARS_FLAGS) . '</a> ' . htmlspecialchars($tEditor->getAuthor(), HTML_SPECIAL_CHARS_FLAGS);
+				echo '<div id="taxonDiv"><a href="../index.php?taxon=' . htmlspecialchars($tEditor->getTid(), HTML_SPECIAL_CHARS_FLAGS) . '">View Public Taxon Profile</a> ';
```

Comment:
This is showing up really huge. Can we make it much smaller
![image](https://github.com/BioKIC/Symbiota/assets/16688784/e2d5eb20-a6dc-42a5-b48b-05f07cefb890)
like on the tpeditor page?
![image](https://github.com/BioKIC/Symbiota/assets/16688784/a156120b-eda9-4fb8-ba89-b5fab4939c84)

Reply:
Fixed in f3dbcc6bd.